dist/video-stream-merger.js

```diff
@@ -4,11 +4,22 @@
 module.exports = VideoStreamMerger
 
 window.AudioContext = window.AudioContext || window.webkitAudioContext
+var audioSupport = !!(AudioContext && (new AudioContext()).createMediaStreamDestination)
+var canvasSupport = !!document.createElement('canvas').captureStream
+var supported = audioSupport && canvasSupport
 
 function VideoStreamMerger (opts) {
   var self = this
   if (!(self instanceof VideoStreamMerger)) return new VideoStreamMerger(opts)
 
+  self.supported = supported
+
+  if (!supported) {
+    self._fallbackStream = null
+    self.started = false
+    self.result = null
+    return
```

Comment:
Please throw an error here instead of returning, so it doesn't fail silently.

`throw new Error('Unsupported browser')`